extensions/oidc-client/runtime/src/main/java/io/quarkus/oidc/client/runtime/OidcClientRecorder.java

```diff
@@ -122,41 +123,53 @@ protected static OidcClient createOidcClient(OidcClientConfig oidcConfig, String
             tokenRequestUriUni = discoverTokenRequestUri(client, authServerUri.toString(), oidcConfig);
         }
         return tokenRequestUriUni.onItemOrFailure()
-                .transform(new BiFunction<String, Throwable, OidcClient>() {
+                .transform((tokenRequestUri, t) -> {
```

Comment:
@mkowa42 I'd prefer to keep the original more verbose form for now - as it is hard to see what has changed - I'll be fine with this simplification though in a follow up PR :-)

Reply:
Reverted